draft-ietf-quic-http.md

```diff
@@ -496,111 +496,96 @@ HEADERS frames can only be sent on request / push streams.
   : A variable-length integer that identifies the element being prioritized.
     Depending on the value of Prioritized Type, this contains the Stream ID of a
-    request stream, the Push ID of a promised resource, a Placeholder ID of a
-    placeholder, or is absent.
-
-  Element Dependency ID:
-  : A variable-length integer that identifies the element on which a dependency
-    is being expressed. Depending on the value of Dependency Type, this contains
-    the Stream ID of a request stream, the Push ID of a promised resource, the
-    Placeholder ID of a placeholder, or is absent.  For details of
-    dependencies, see {{priority}} and {{!RFC7540}}, Section 5.3.
-
-  Weight:
-  : An unsigned 8-bit integer representing a priority weight for the prioritized
-    element (see {{!RFC7540}}, Section 5.3). Add one to the value to obtain a
-    weight between 1 and 256.
-
-The values for the Prioritized Element Type ({{prioritized-element-types}}) and
-Element Dependency Type ({{element-dependency-types}}) imply the interpretation
-of the associated Element ID fields.
+    request stream or the Push ID of a promised resource.
+
+  Encoded Priority:
+  : An unsigned variable-length integer representing a priority for the 
+    prioritized element. When deciding between a set of available resources to
+    serve, a server SHOULD strictly prefer sending resources with a higher 
+    priority to those of a lower priority. If two or more  available resources 
+    have equal priority, they MAY be interleved.
+    
+    
+The Encoded Priority can be interpreted as a fixed point number with the radix 
+after the first six bits (the remaining bits after the 2 bit length encoding). 
+For intengers of length one the value is their normal value. For lengths greater 
+than one, bytes after the first byte should be interpreted as coming after the 
+radix point and decreasing in signifigance. For translation into comparable 64 
+bit integers, an implementation could interpreted the priorities internally as 
+the integer value left shifted by 64 minus the length of the field in bits. 
+
+| Value | Encoded |  Wire  | 64-bit hex representation |
+|-------|---------|--------|---------------------------|
+|  2    |  0x02   |  0x02  |    0x0200000000000000     |
+| 2.5   | 0x0280  | 0x4280 |    0x0280000000000000     |
+| 1.25  | 0x0140  | 0x4140 |    0x0140000000000000     |
+
+A client that has a small set of priority levels but would like to have the
+delivery of resources not interlieved can achieve this by reserving the top X bits
+for those priorities and the bottom 6-X bits for differentiating the resources and
+not use more than one byte. So, if the implementation had four priority buckets, 
+they could have 16 concurrent resources within a given priority bucket without 
+risking any of them being delivered interleaved. If a priority bucket has more than
+16 outstanding requests the client can use an addtional byte and handle an 
+additional 4080 requests.
```

Comment:
@ianswett does this get the concept across clearly?

Reply:
Works for me